RFC-004-revising-licenses-page.md

```diff
@@ -0,0 +1,118 @@
+
+On a personal level, I don't release my likeness to public use, and the current website license terms makes me wary of continuing with the project. I used to work in games (and still do at times), and I've had too many death threats (honestly, one such threat is too many) and have seen too many friends be impersonated to want to give bad actions a contractual right to use my likeness however they like. (Bad actors will still use likenesses, but at least they can't defend their actions by citing a license that allows them to use one's info.)
+
+### Blocking
+
+[RFC002 (Procope/blog & video)](https://github.com/thegooddocsproject/request-for-comment/pull/3) is blocked by this proposal.
+
+## Proposal
+
+I'd like to shift the burden of licensing to the individual elements, rather than have a pure blanket. That means:
+
+- Our products (templates, doc tools, etc) have the BSD0 license in their readme or license text files, and not rely on the license website page for that notice
+- For blog posts etc, post pages would default to have "`This blog post's original text is (c) [year] [author name], and licensed under CC BY 4.0`". Currently, the blog is set up to allow for this default, with other changes allowed. Ryan and Bryan worked together to make sure this could be modular.
+- "Original text" covers not opening any quoted text (like in referencing a book's text) as being CC, and doesn't open any stock images from being CC. If a custom image is created by the author, we can add that to the CC license, which is part of why Ryan and Bryan worked to make the license field modular.
+- For everything else, we leave the material either unlicensed or under whatever licenses already exist for that material. (See What's currently CC BY content below.)
+
+Since the licenses page wouldn't be the declaration of broad license anymore, it would become the page that explains what licences are used: BSD0 for our products, CC BY for "various pages" on the site. The licenses page wouldn't be the *primary authority* for this information, but instead a *single page to set expectations* for contributors and consumers regarding using elements of this project.
+
+### Keeping it simple
+
+Cameron mentioned wanting to keep this scheme simple. Here's how we keep to minimal labor:
+
+- There are defaults, and we assume them. I've proposed those defaults above.
+- For the generated content (like blog posts), the generator adds the license text. No need to sweat it.
+- For products, they should already have licenses as part of the pages, because once downloaded or forked the license needs to be attached rather than disconnected on an external site
+- Having the license attached to the website content means readers see the license alongside the text, and don't have to wonder or ask or hunt around for licensing info
```

Comment:
praise: I think having the license easily findable on each post is a great improvement!

Reply:
praise: +1